### DISCUSSION TITLE
redirection des enregistrerments d'essai de cantines dans démo?

### DISCUSSION THREAD

**Discussion by valeriemerle** [betagouv/ma-cantine#1036] [Idées]
Je trouve régulièrement des "fausses" cantines dans admin. Que doit-on/peut-on en faire? les supprimer, les basculer vers démo?

**Comment by hfroot** [comment]
Perso je pense que c'est mieux de contacter l'utilisateur et leur demander de supprimer la cantine - comme ça il est conscient du changement et la fiabilité de données reste la responsabilité d'utilisateur

**Reply by valeriemerle** [reply]
Certaines sont les nôtres! Pour les autres je les liste dans un fichier et j'envoie un mail copie à contact pour leur demander le statut de leur compte en les engageant à aller plutot sur demo par ex?

**Comment by alemangui** [comment]
On peut les contacter si on pense que ce sont des fausses cantines, mais malheureusement tant qu'on n'ait pas une procédure de validation pour s'assurer qu'un utilisateur est bien le gestionnaire d'une cantine, des erreurs et comptes test seront toujours là. C'est d'ailleurs le cas pour tous les systèmes qui permettent une création de compte publique.
